pallets/pallet-bonded-coins/src/mock.rs

```diff
@@ -11,3 +50,305 @@ pub fn assert_relative_eq(target: Float, expected: Float, epsilon: Float) {
 		target
 	);
 }
+
+pub(crate) fn get_linear_bonding_curve() -> Curve<Float> {
+	let m = Float::from_num(0);
+	let n = Float::from_num(2);
+	let o = Float::from_num(3);
+	Curve::Polynomial(PolynomialParameters { m, n, o })
+}
+
+pub(crate) fn get_linear_bonding_curve_input() -> CurveInput<FloatInput> {
+	let m = FloatInput::from_num(0);
+	let n = FloatInput::from_num(2);
+	let o = FloatInput::from_num(3);
+	CurveInput::Polynomial(PolynomialParametersInput { m, n, o })
+}
+
+pub(crate) fn calculate_pool_id(currencies: Vec<AssetId>) -> AccountId {
+	AccountId::from(currencies.blake2_256())
+}
+
+#[cfg(test)]
```

Comment:
Aren't all the functions above also used just for tests? Why only the runtime module is test-gated?

Reply:
The whole mock is test-gated. I removed the mod in: [cb1bdd5](https://github.com/KILTprotocol/kilt-node/pull/780/commits/cb1bdd59504be003f6f6c19e3472a6337dd5af7e)